hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/snapshot/SnapshotDiffManager.java

```diff
@@ -1010,6 +1066,11 @@ private long generateDiffReport(
       try (ClosableIterator<byte[]>
                objectIdsIterator = objectIDsToCheck.iterator()) {
         while (objectIdsIterator.hasNext()) {
+          if (snapDiffJobTable.get(jobKey).getStatus()
```

Comment:
> To avoid NPE.

Yes, the job might have failed or being rejected and got removed from the table.

> Should we check after like every 100 entries?

Sure, I'll fix that.